src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -318,37 +349,49 @@ class DialogmoteService(
         connection: Connection,
         arbeidstakerUuid: UUID,
         arbeidstakerId: Int,
+        arbeidsgiverId: Int,
         arbeidstakerPersonIdent: PersonIdentNumber,
-        pdf: ByteArray,
+        pdfArbeidstaker: ByteArray,
+        pdfArbeidsgiver: ByteArray,
         narmesteLeder: NarmesteLederDTO,
         varselType: MotedeltakerVarselType,
-        fritekst: String = "",
-        innkalling: List<DocumentComponentDTO> = emptyList(),
+        fritekstArbeidstaker: String = "",
+        fritekstArbeidsgiver: String = "",
+        innkallingArbeidstaker: List<DocumentComponentDTO> = emptyList(),
+        innkallingArbeidsgiver: List<DocumentComponentDTO> = emptyList(),
     ) {
-        val (_, varselId) = connection.createMotedeltakerVarselArbeidstaker(
+        val (_, varselArbeidstakerId) = connection.createMotedeltakerVarselArbeidstaker(
             commit = false,
             motedeltakerArbeidstakerId = arbeidstakerId,
             status = "OK",
             varselType = varselType,
             digitalt = true,
-            pdf = pdf,
-            fritekst = fritekst,
-            document = innkalling,
+            pdf = pdfArbeidstaker,
+            fritekst = fritekstArbeidstaker,
+            document = innkallingArbeidstaker,
+        )
+        connection.createMotedeltakerVarselArbeidsgiver(
+            commit = false,
+            motedeltakerArbeidsgiverId = arbeidsgiverId,
+            status = "OK",
+            varselType = varselType,
+            pdf = pdfArbeidsgiver,
+            fritekst = fritekstArbeidsgiver,
+            document = innkallingArbeidsgiver,
         )
         val now = LocalDateTime.now()
         arbeidstakerVarselService.sendVarsel(
             createdAt = now,
             personIdent = arbeidstakerPersonIdent,
             type = varselType,
             motedeltakerArbeidstakerUuid = arbeidstakerUuid,
-            varselUuid = varselId,
+            varselUuid = varselArbeidstakerId,
         )
         narmesteLederVarselService.sendVarsel(
             createdAt = now,
             narmesteLeder = narmesteLeder,
             varseltype = varselType
         )
```

Comment:
Slik koden er nå så vil følgende skjer dersom `narmesteLederVarselService.sendVarsel` feiler:
Vi vil ha opprettet Oppgave på brukernotifikasjon-topic og det vil komme opp en Oppgave på DittNAV for arbeidstaker, men det vil ikke være noe brev og lese/ferdigstille.

Trenger ikke ta høyde for det i denne PR'en, men vi burde få lagt til noe som håndtere dette senere.

Reply:
Ja, vi får diskutere hvordan vi skal ta høyde for det problemet.